src/main/kotlin/com/github/devcordde/devcordbot/core/JavaDocManager.kt

```diff
@@ -31,34 +28,15 @@ import org.jsoup.nodes.Document
 object JavaDocManager {
```

Comment:
Why does this still exist? You removed the Javadoc command, didn't you?

Reply:
auto-help still calls it it will be removed when autohelp gets updates